admin/sql/updates/20160514-mbs-8287-a.sql

```diff
@@ -0,0 +1,46 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CREATE TABLE deleted_entity (
+    gid UUID NOT NULL,
+    data JSONB NOT NULL,
+    deleted_at timestamptz NOT NULL DEFAULT now()
+);
+
+
+INSERT INTO deleted_entity (gid, deleted_at, data)
+SELECT gid, deleted_at,
+       jsonb_object('{last_known_name, last_known_comment, entity_gid, entity_type}',
+                ARRAY[last_known_name, last_known_comment, gid::text, 'artist']) AS data
+  FROM artist_deletion;
```

Comment:
No need to throw out the existing data about deleted artists, labels, and series.